test/000_fullchain.test.ts

```diff
@@ -297,10 +298,125 @@ describe('Integration tests', function () {
                     frozenChanges: expectedFrozenChanges,
                 });
                 for (let i = 0; i < workerNumber; i++) {
-                    if (workerNumber == 1) {
-                        expect(await iexecPoco.viewScore(workers[i].address)).to.be.equal(0);
-                    }
+                    expect(await iexecPoco.viewScore(workers[i].address)).to.be.equal(
+                        workerNumber == 1 ? 0 : 1,
+                    );
+                }
+            });
+        }
+        for (let workerNumber = 4; workerNumber < 6; workerNumber++) {
```

Comment:
I think we can keep only 1 test with either 4 or 5 workers and 1 bad worker. We can even do 5 workers with 2 bad workers. But I don't see the benefit of doing more.

Reply:
Okay I will keep 1 test => 4 good 1 bad

Reply:
Updated [here](https://github.com/iExecBlockchainComputing/PoCo/pull/160/commits/982f21c5ce13fddd16df334507bd33eef881b229)